diff --git a/meta-imx-sdk/conf/distro/include/fsl-imx-preferred-env.inc b/meta-imx-sdk/conf/distro/include/fsl-imx-preferred-env.inc
index 2ba50abbc7..8774f0531b 100644
--- a/meta-imx-sdk/conf/distro/include/fsl-imx-preferred-env.inc
+++ b/meta-imx-sdk/conf/distro/include/fsl-imx-preferred-env.inc
@@ -1,5 +1,5 @@
-PREFERRED_PROVIDER_virtual/bootloader:imx-nxp-bsp = "u-boot-imx"
-PREFERRED_PROVIDER_virtual/kernel:imx-nxp-bsp = "linux-imx"
+#PREFERRED_PROVIDER_virtual/bootloader:imx-nxp-bsp = "u-boot-imx"
+#PREFERRED_PROVIDER_virtual/kernel:imx-nxp-bsp = "linux-imx"
 PREFERRED_PROVIDER_linux-imx-mfgtool = "linux-imx-mfgtool"

 PREFERRED_VERSION_gstreamer1.0              = "1.24.0.imx"
